### PR TITLE
Updated latest code with few configuration changes.

### PR DESCRIPTION
**Pull request by arun-msft**
As per our discussion modified folder name and readme file.

### CODE REVIEW

File: TeamsSampleHub/README.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
-Microsoft Teams HOME app overview
+Microsoft Teams Sample hub app overview
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:1]: Remove "app" across the doc. it should be "Teams Sample Hub", in this case Teams Sample Hub overview

File: TeamsSampleHub/README.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 =====================================
 
-The Teams Branded Experience (aka HOME app), together with custom
+The Teams Branded Experience (aka Sample hub app), together with custom
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:4]: aka Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 line-of-business applications, enable you to quickly present the "art of
 the possible" by showcasing the Teams platform and articulating the
 value that it can deliver. 
 
-The HOME app is a personal-branded landing experience that is pinned to
+The Sample hub app is a personal-branded landing experience that is pinned to
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:9]: remove app

File: TeamsSampleHub/README.md [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
 the Teams mobile app, allowing your organization to provide consolidated
 information and the functionality of important first-party and
-line-of-business apps to firstline worker employees. HOME can leverage
+line-of-business apps to firstline worker employees. Sample hub can leverage
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:12]: Sample Hub

File: TeamsSampleHub/README.md [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
 multiple technologies and capabilities such as the SharePoint Framework
 in Teams (SPFx), Microsoft Graph APIs, PowerApps, Web
 Apps/HTML/JavaScript, and mobile app pinning to highlight a branded
 experience for any user.
 
-The HOME app was primarily designed and built as a mobile app and is
+The Sample hub app was primarily designed and built as a mobile app and is
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:18]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 6]
```diff
@@ -1,35 +1,35 @@
...
 currently not optimized for desktop clients.
 
-Home app scenarios
+Sample hub app scenarios
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:21]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 7]
```diff
@@ -1,35 +1,35 @@
...
 =================
 
 
 <table>
     <tbody>
         <tr>
             <td class="col-md-8" style="color:blue;" align="center"><b>On shift scenarios</b></td>
-            <td rowspan="6"><img src="https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/1.png" alt="Home App Screen" style="width: 100%;"></td>
+            <td rowspan="6"><img src="https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/1.png" alt="Sample hub Screen" style="width: 100%;"></td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:29]: Sample Hub

File: TeamsSampleHub/README.md [hunk 8]
```diff
@@ -1,35 +1,35 @@
...
          </tr>
        <tr>
-            <td>Employees can view their current shift schedule and current assigned tasks under the Home tab.</td>
+            <td>Employees can view their current shift schedule and current assigned tasks under the Hub tab.</td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:32]: Looks good

File: TeamsSampleHub/README.md [hunk 9]
```diff
@@ -44,22 +44,22 @@ Home app scenarios
             <td>Employees can communicate and greet new team members using Teams 1:1 chat capability. </td>
         </tr>
        <tr>
-            <td>Employees can access other apps such as PayStubs and Rewards through the HOME application. </td>
-           <td><img src="https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/2.png" alt="Home App Screen"></td>
+            <td>Employees can access other apps such as PayStubs and Rewards through the Sample hub application. </td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:47]: Sample Hub

File: TeamsSampleHub/README.md [hunk 10]
```diff
@@ -44,22 +44,22 @@ Home app scenarios
...
+           <td><img src="https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/2.png" alt="Sample hub App Screen"></td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:48]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 11]
```diff
@@ -44,22 +44,22 @@ Home app scenarios
...
         </tr>
          <tr>
             <th colspan="2"></th>
          </tr>
         <tr>
             <td align="center"><b>Off shift scenarios</b></td>
-            <td rowspan="2"><img src="https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/3.png" alt="Optional Title"></td>
+            <td rowspan="2"><img src="https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/3.png" alt="Optional Title"></td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:55]: looks good

File: TeamsSampleHub/README.md [hunk 12]
```diff
@@ -44,22 +44,22 @@ Home app scenarios
...
          </tr>
        <tr>
             <td>Off shift employees can view their upcoming shift schedule. Off shift employees cannot view and access tasks.</td>
         </tr>
        <tr>
            <td>Off shifts employees do not have access to Learning tab.</td>
-           <td><img src="https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/4.png" alt="Optional Title"></td>
+           <td><img src="https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/4.png" alt="Optional Title"></td>
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:62]: looks good

File: TeamsSampleHub/README.md [hunk 13]
```diff
@@ -103,12 +103,12 @@ Prerequisites
...
 5.  Install your desired custom apps.
 
-![note](https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/6.PNG)
+![note](https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/6.PNG)
 
 Authentication and Single Sign on (SSO)
 ---------------------------------------
 
-The HOME app authentication flow is based on user profile information
+The Sample hub app authentication flow is based on user profile information
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:111]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 14]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
 Capabilities overview
 ---------------------
 
-1.  HOME app is an ASP.NET Core MVC application.
+1.  Sample hub app is an ASP.NET Core MVC application.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:126]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 15]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
 
 2.  Access employee Shifts schedules using [Microsoft Teams Shifts Graph API integration](https://docs.microsoft.com/en-us/graph/api/schedule-list-shifts?view=graph-rest-1.0&tabs=http).
 
 3.  Display employee tasks using [Microsoft Planner Tasks Graph API
     integration](https://docs.microsoft.com/en-us/graph/api/planneruser-list-tasks?view=graph-rest-1.0&tabs=http).
 
-4.  Customize HOME app to create a tailored experience. For example:
+4.  Customize Sample hub app to create a tailored experience. For example:
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:133]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 16]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
 
-    -   The announcement feature of the HOME app can be integrated with
+    -   The announcement feature of the Sample hub app can be integrated with
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:135]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 17]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
         our [Company Communicator app
         template](https://docs.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#company-communicator)
         to share your company's the information and notifications within
-        the HOME app.
+        the Sample hub app.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:139]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 18]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
 
     -   [Create Power App
         applications](https://docs.microsoft.com/en-us/learn/modules/get-started-with-powerapps/5-powerapps-create-first)
         and
         [deep-link](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links)
-        to the HOME app.
+        to the Sample hub app.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:145]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 19]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
 
     -   [Build SharePoint Framework application in
         Teams](https://aka.ms/spfx-teams) such as a News app and
         [deep-link](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links)
-        to the HOME app.
+        to the Sample hub app.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:150]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 20]
```diff
@@ -123,39 +123,39 @@ about authentication for Microsoft Tabs and SSO in Teams.
...
 
 5.  Retrieve a list of team members in the same shift by using the
     [Microsoft Graph List Members
     API](https://docs.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0&tabs=http).
 
 6.  Enhance the user chat experience by creating [deep
     links](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/deep-links#generating-a-deep-link-to-a-chat)
-    to private chats within the HOME app.
+    to private chats within the Sample hub app.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:158]: Sample Hub, remove app

File: TeamsSampleHub/README.md [hunk 21]
```diff
@@ -167,13 +167,13 @@ Capabilities overview
 Deployment
 ----------
 
-### Register your HOME app with Azure AD
+### Register your Sample hub app with Azure AD
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:170]: Sample Hub,

File: TeamsSampleHub/README.md [hunk 22]
```diff
@@ -167,13 +167,13 @@ Capabilities overview
...
 
 -   Sign in to the [Azure portal.](https://portal.azure.com)
 
 -   If your account gives you access to more than one tenant, select your account in the upper right corner. Set your portal session to your desired Azure AD tenant
 
--   Under Azure services on the Home page, select App registrations.
+-   Under Azure services on the Sample hub page, select App registrations.
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:176]: Sample Hub,

File: TeamsSampleHub/README.md [hunk 23]
```diff
@@ -221,9 +221,9 @@ Graph notifications.
 
 Please make sure to grant the admin consent for the required permissions.
 
-### Publish your HOME app to your Azure subscription using Visual Studio 
+### Publish your Sample hub app to your Azure subscription using Visual Studio
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:224]: Sample Hub

File: TeamsSampleHub/README.md [hunk 24]
```diff
@@ -221,9 +221,9 @@ Graph notifications.
 
 Please make sure to grant the admin consent for the required permissions.
 
-### Publish your HOME app to your Azure subscription using Visual Studio 
+### Publish your Sample hub app to your Azure subscription using Visual Studio 
 
--   Download the HOME code from the repository and open in Visual studio
+-   Download the Sample hub code from the repository and open in Visual studio
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:226]: Sample Hub

File: TeamsSampleHub/README.md [hunk 25]
```diff
@@ -247,20 +247,20 @@ Please make sure to grant the admin consent for the required permissions.
         3. If you have SharePoint Framework app then provide the SharePoint client Id and List Id.        
 -   When ready, select Create.
 
-### Create a HOME Teams app manifest
+### Create a Sample hub Teams app manifest
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:250]: Sample Hub

File: TeamsSampleHub/README.md [hunk 26]
```diff
@@ -247,20 +247,20 @@ Please make sure to grant the admin consent for the required permissions.
...
 
-[Create an app package for the HOME
+[Create an app package for the Sample hub
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:252]: Sample Hub

File: TeamsSampleHub/README.md [hunk 27]
```diff
@@ -247,20 +247,20 @@ Please make sure to grant the admin consent for the required permissions.
...
 Apps in Teams are defined by an app manifest JSON file and bundled in an
 app package with their icons. You\'ll need an app package to upload and
 install your app in Teams, and to publish to either your Line of
 Business app catalog or to AppSource.
 
-![Note](https://github.com/msft-teams/tools/blob/master/HOMEapp/Brandhome/Source/wwwroot/images/readmeimages/7.PNG)
+![Note](https://github.com/msft-teams/tools/blob/master/TeamsSampleHub/SampleHub/Source/wwwroot/images/readmeimages/7.PNG)
 
-### Install the HOME Teams app manifest into Teams
+### Install the Sample hub Teams app manifest into Teams
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:261]: Sample Hub

File: TeamsSampleHub/README.md [hunk 28]
```diff
@@ -247,20 +247,20 @@ Please make sure to grant the admin consent for the required permissions.
...
 
-Install the HOME app manifest into Teams. [Learn how to distribute Microsoft Teams
+Install the Sample hub app manifest into Teams. [Learn how to distribute Microsoft Teams
```
Comment (aamini-ms) [inline TeamsSampleHub/README.md:263]: Sample Hub

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 29]
```diff
@@ -12,12 +12,12 @@
     "mpnId": "{your mpnID}"
   },
   "name": {
-    "short": "HOME App",
-    "full": "HOME App"
+    "short": "Sample hub",
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:15]: Sample Hub

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 30]
```diff
@@ -12,12 +12,12 @@
...
+    "full": "Sample hub"
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:16]: Sample Hub

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 31]
```diff
@@ -12,12 +12,12 @@
...
   },
   "description": {
-    "short": "HOME app",
-    "full": "The HOME app is a personal-branded landing experience that is pinned to the Teams mobile app, allowing your organization to provide consolidated information and the functionality of important first-party and line-of-business apps to firstline worker employees."
+    "short": "Sample hub app",
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:19]: Sample Hub

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 32]
```diff
@@ -12,12 +12,12 @@
...
+    "full": "The Sample hub app is a personal-branded landing experience that is pinned to the Teams mobile app, allowing your organization to provide consolidated information and the functionality of important first-party and line-of-business apps to firstline worker employees."
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:20]: Sample Hub, remove app

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 33]
```diff
@@ -26,8 +26,8 @@
   "accentColor": "#60A18E",
   "staticTabs": [
     {
-      "entityId": "Home",
-      "name": "Home",
+      "entityId": "Sample hub",
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:29]: Sample Hub

File: TeamsSampleHub/SampleHub/Manifest/manifest.json [hunk 34]
```diff
@@ -26,8 +26,8 @@
...
+      "name": "Sample hub",
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Manifest/manifest.json:30]: Sample Hub

File: TeamsSampleHub/SampleHub/Source/Controllers/ApplicationConfigurationsController.cs [hunk 35]
```diff
@@ -36,7 +36,7 @@ public ApplicationConfigurationsController(IConfiguration configuration, Telemet
         }
 
         /// <summary>
-        /// Home Tab
+        /// Sample hub Tab
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Source/Controllers/ApplicationConfigurationsController.cs:39]: Sample Hub

File: TeamsSampleHub/SampleHub/Source/Views/ApplicationConfigurations/Index.cshtml [hunk 36]
```diff
@@ -20,7 +20,7 @@
     </script>
     <script src="~/js/Index.js"></script>
     <script src="~/js/home.js"></script>
-    <title>Home App</title>
+    <title>Sample hub</title>
```
Comment (aamini-ms) [inline TeamsSampleHub/SampleHub/Source/Views/ApplicationConfigurations/Index.cshtml:23]: Sample Hub

File: TeamsSampleHub/README.md (hunk 11 above)
Comment (arun-msft) [reply TeamsSampleHub/README.md:55]: Updated app name as suggested.

File: TeamsSampleHub/SampleHub/Source/Views/ApplicationConfigurations/Index.cshtml (hunk 36 above)
Comment (arun-msft) [reply TeamsSampleHub/SampleHub/Source/Views/ApplicationConfigurations/Index.cshtml]: Updated app name as suggested.

File: TeamsSampleHub/SampleHub/Source/Controllers/ApplicationConfigurationsController.cs (hunk 35 above)
Comment (arun-msft) [reply TeamsSampleHub/SampleHub/Source/Controllers/ApplicationConfigurationsController.cs]: Updated app name as suggested.

**Comment by arun-msft** [comment]
Updated configuration files and content.

**Pull request merged**